base/deprecated.jl

```diff
@@ -1855,6 +1855,12 @@ end
     nothing
 end
 
+@deprecate find(x::Number)            find(x->x!=0, x)
```

Comment:
Should we encourage `find(!iszero, x)` for this?

Reply:
Ah yes, for `Number` I guess we should.